project/Jdk9.scala

```diff
@@ -15,14 +15,11 @@ import sbt.Keys._
 import sbt._
 
 object Jdk9 extends AutoPlugin {
-  import JdkOptions.notOnJdk8
   import JdkOptions.JavaVersion._
 
-  // The version 9 is special for any Java versions >= 9
-  // and the version 11 is special for any Java versions >= 11
-  // and the version 17 is special for any Java versions >= 17
+  // The version 17 is special for any Java versions >= 17
   // and the version 21 is special for any Java versions >= 21
-  private val supportedJavaLTSVersions = List("9", "11", "17", "21")
+  private val supportedJavaLTSVersions = List("17", "21")
 
   lazy val CompileJdk9 = config("CompileJdk9").extend(Compile)
```

Comment:
I suppose we can remove this whole concept - but perhaps that's for a subsequent PR